libs/gi/page-home/src/InventoryCard.tsx

```diff
@@ -183,6 +204,67 @@ export default function InventoryCard() {
             ))}
           </CardActionArea>
         </CardThemed>
+        <CardThemed bgt="light">
+          <CardActionArea
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              p: 2,
+              gap: 1,
+              flexWrap: 'wrap',
+            }}
+            component={RouterLink}
+            to="/teams"
+          >
+            <Chip
+              sx={{ flexGrow: 1, cursor: 'pointer' }}
+              color={numTeams ? 'success' : 'secondary'}
+              icon={<TeamIcon />}
+              label={
+                <strong>
+                  {t(`ui:tabs.teams`)} {numTeams}
+                </strong>
+              }
+            />
+            <Chip
+              sx={{ flexGrow: 1, cursor: 'pointer' }}
+              color={numLoadout ? 'success' : 'secondary'}
+              icon={<LoadoutIcon />}
+              label={
+                <strong>
+                  {t(`ui:tabs.loadouts`)} {numLoadout}
+                </strong>
+              }
+            />
+            <Chip
+              sx={{ flexGrow: 1, cursor: 'pointer' }}
+              color={numBuilds ? 'success' : 'secondary'}
```

Comment:
Should we start using less colors here? Or are you gonna do that in a different pr

Reply:
probably a different PR, will def need more design guidance before pulling the trigger on that.